src/ess_rhat.jl

```diff
@@ -239,19 +257,184 @@ The ESS and ``\\widehat{R}`` values can be computed for the following estimators
     doi: [10.1214/20-BA1221](https://doi.org/10.1214/20-BA1221)
     arXiv: [1903.08008](https://arxiv.org/abs/1903.08008)
 """
-function ess_rhat(samples::AbstractArray{<:Union{Missing,Real},3}; kwargs...)
-    return ess_rhat(Statistics.mean, samples; kwargs...)
+function ess(samples::AbstractArray{<:Union{Missing,Real},3}; kind=:bulk, kwargs...)
+    if kind === :bulk
+        return _ess(Val(:bulk), samples; kwargs...)
```

Comment:
Maybe add a comment to explain why these branches exist and we not just use `Val(kind)`?